src/components/filter/Filter.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {connect} from "react-redux";
+import {selectFilterAction} from "../../actions";
+import styles from "../../css/styles.module.css";
+
+const Filter = props => (
```

Comment:
Please keep your code formatted. Previously you have used parentheses. Just use it everywhere or don't use at all